.circleci/config.yml

```diff
@@ -343,27 +344,124 @@ jobs:
       # stabilize benchmark runs. We should eventually have two different 
       # set of binaries. 
       - run:
-          name: "Run Benchmarks - Baseline"
+          name: "Run Benchmarks - Round 1"
           command: |
             ./scripts/benchmark-runner.py run \
                 --bm_estimate_time \
                 --bm_max_secs 10 \
                 --bm_max_trials 10000 \
                 --output_path ${BASELINE_OUTPUT_PATH}
+
+            ./scripts/benchmark-runner.py run \
+                --bm_estimate_time \
+                --bm_max_secs 10 \
+                --bm_max_trials 10000 \
+                --output_path ${TARGET_OUTPUT_PATH}
+
+            ./scripts/benchmark-runner.py compare \
+                --baseline_path ${BASELINE_OUTPUT_PATH} \
+                --target_path ${TARGET_OUTPUT_PATH} \
+                --rerun_json_output ${RERUN_JSON_OUTPUT} || true
+      - store_artifacts:
+          path: ${BASELINE_OUTPUT_PATH}
+      - store_artifacts:
+          path: ${TARGET_OUTPUT_PATH}
+      - run:
+          name: "Run Benchmarks - Round 2"
```

Comment:
To cut down on duplicated code, maybe consider making this block a `command` at the top of this file and calling the command four more times down here? You could include a parameter to change the `name` each time if you'd still like to keep the different names.